src/components/custom/signup.js

```diff
@@ -91,7 +91,9 @@ class Signup extends Component {
     }
 
     getVerifyEmailRequest = (email) => {
-        const utm_data = TrafficSource.getData()
+        const utm_data_cookies = new CookieStorage('utm_data')
+        const utm_data_value = utm_data_cookies.get('utm_data')
+        const utm_data = utm_data_value ? getUTMData(utm_data_value) : {}
```

Comment:
can you change these behavior as well to get utm term and utm content, and we dont need to put in utm referrer and direct 🙏 , can straightforward do check if its there then add, if not, we can do nothing